mon-pix/tests/acceptance/terms-of-service-test.js

```diff
@@ -20,14 +19,24 @@ describe('Acceptance | terms-of-service', function() {
     });
   });
 
+  describe('When user log in and must validate Pix latest terms of service', async function() {
```

Comment:
Je propose, comme titre de commit
Add tests in terms-of-service  => Add acceptance tests for terms of service validation